app/scripts/components/contentcreator.jsx

```diff
@@ -47,10 +47,10 @@ var ContentCreator = React.createClass({
 
         // populate content with appropriate metadata
         if (this.props.forComment) {
-            content["post"] = this.props.post;
+            content.post = this.props.post;
             PostActions.newComment(content);
         } else {
-            content["comments"] = [];
+            content.post = [];
```

Comment:
Nitpicky, but this should probably initialize to any empty object instead of an array, right?